assets/tiled-project/tilemaps/backCave.tmx

```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <map version="1.5" tiledversion="1.6.0" orientation="orthogonal" renderorder="right-down" width="14" height="12" tilewidth="32" tileheight="32" infinite="0" nextlayerid="26" nextobjectid="9">
  <editorsettings>
-  <export target="../../exported-maps/backCave.json" format="json"/>
+  <export format="json"/>
```

Comment:
Not sure why this line went away - i think without it autoexport will not work. Not a biggie though - can be fixed when the location is worked on later